lib/src/requester/request/subscribe.dart

```diff
@@ -15,8 +15,8 @@ class ReqSubscribeListener implements StreamSubscription {
     return null;
   }
 
-  // TODO: define a custom class to replace StreamSubscription
-  Future asFuture([futureValue]) {
+  // TODO: define a custom class to replace StreamSubscription.
+  Future<dynamic/*=E*/> asFuture/*<E>*/([/*=E*/ dynamic futureValue]) {
```

Comment:
We have to use the comment syntax because there are instances still using old versions of Dart, but this satisfies the Dart analyzer. Dart 2.0 might not treat the comments the same way, we might want to look at that in the future.